week2/btree.py

```diff
@@ -0,0 +1,17 @@
+class BinaryTree:
+	'''Data structure, consists of root and two children which are trees as well (can be empty)
+		self.root: root element value
+		self.left: left child value
+		self.right: right child value
+		addRightChild: adds right child tree
```

Comment:
nit: While fields are well documents as part of the class definition, methods are better documented in the definition of the method itself.